server/app/controllers/HomeController.java

```diff
@@ -62,6 +67,12 @@ public CompletionStage<Result> index(Http.Request request) {
                 // If the applicant has not yet set their preferred language, redirect to
                 // the information controller to ask for preferred language.
                 ApplicantData data = applicant.getApplicantData();
+                if (langs.availables().size() <= 1 && !data.hasPreferredLocale()) {
+                  data.setPreferredLocale(
+                      langs.availables().isEmpty()
+                          ? Locale.US
+                          : langs.availables().get(0).toLocale());
+                }
```

Comment:
Do we have a util file where we can share "setPreferredLocale" method with the other controller?

Reply:
added util file.